backend/src/main/java/de/neuefische/readandmeet/backend/service/MeetingService.java

```diff
@@ -25,26 +24,27 @@ public List<Meeting> list() {
         return this.meetingRepo.findAll();
     }
 
-    public Meeting add(MeetingWithoutId m) {
+    public Meeting add(MeetingWithoutId meetingWithoutId) {
         String id = uuIdService.getRandomId();
-        Meeting meeting = new Meeting(id, m.getTitle(), m.getDate(), m.getLocation());
+        Meeting meeting = new Meeting(id, meetingWithoutId.getTitle(), meetingWithoutId.getDate(), meetingWithoutId.getLocation(), meetingWithoutId.getBook());
         return this.meetingRepo.insert(meeting);
     }
+
     public Meeting getDetails(String id) {
-        return this.meetingRepo.findById(id).orElseThrow(() -> new NoSuchElementException(id));
+        return this.meetingRepo.findById(id).orElseThrow(() -> new NoSuchMeetingException(id));
     }
 
     public void delete(String id) {
         Meeting meeting = this.meetingRepo.findById(id)
-                .orElseThrow(() -> new NoSuchElementException("Meeting with ID " + id + " not found"));
+                .orElseThrow(() -> new NoSuchMeetingException(id));
         this.meetingRepo.delete(meeting);
     }
 
-    public Meeting editMeetingById(MeetingWithoutId m, String id) {
+    public Meeting editMeetingById(MeetingWithoutId meetingWithoutId, String id) {
         Meeting meeting = this.meetingRepo.findById(id)
-                .orElseThrow(() -> new NoSuchElementException("Meeting with ID " + id + " not found"));
+                .orElseThrow(() -> new NoSuchMeetingException(id));
```

Comment:
yes you're right I like this too, but I only did it because sonarcloud marked this as a code smell. I've the same line in the book service class but there it is somehow "ok"